docs/Community/SecurityAdvisoryHandling.md

```diff
@@ -0,0 +1,81 @@
+
+Once a vulnerability related to the Open Enclave SDK is reported to MSRC, the report is forwarded to the committers responsible for reviewing security issues:
+
+- Anand Krishnamoorthi (@anakrish)
+- Ming-wei Shih (@mingweishih)
+- Simon Leet (@codemonkeyleet)
+
+> - [ ] TODO: Formalize SIG-Security and its membership with the CGC.
+
+## Handling Embargoed Vulnerabilities
+
+The security reviewers are responsible for the initial triage of the report. This includes:
+
+- reaching out to the appropriate OE SDK area owners or subject matter experts
+- evaluating the severity of the issue against the bug bar
+- writing up the initial assessment and plan of action for MSRC
+- coordinating with MSRC on additional information, acknowledgment, and public disclosure plans with the reporters of the issue
+
+> - [ ] TODO: Clean up and finalize the security bug triage bar (PR #2634).
+
+The report is treated as under embargo and will only be discussed using private communication channels.
+All individuals entrusted with the information should exercise due diligence in maintaining its confidentiality.
+
+> - [ ] TODO: Formalize the protected channel of communication for discussing the development of embargoed bug fixes (e.g., Protected mailing list, Element private channel).
+
+If OE SDK needs to issue a Security Advisory to address the reported concern, one of the security reviewers will:
+
+1. Create a new GitHub [Security Advisory (SA)](https://github.com/openenclave/openenclave/security/advisories) for OE SDK.
+   1. The description of the SA should accurately capture the nature, severity, and workarounds for the issue.
+      1. It does not need to capture the full technical details of the vulnerability or fix for brevity.
+2. Request a Common Vulnerability Enumeration (CVE) through GitHub.
+   1. This was previously done through MSRC as a CVE Numbering Authority (CNA), but OE has shifted to using GitHub as the CNA as a Linux Foundation project.
+3. Assign the appropriate collaborators to the SA, which includes:
+   1. The assignee(s) responsible for fixing the issue.
+   2. Any area owners or subject matter experts needed for review.
+   3. CI/CD coordinators for private testing and integration with the release. (e.g. @brmclaren).
+4. Create a temporary private fork to implement the code changes as part of the SA.
+   1. The assignee should follow the GitHub instructions and create an `advisory-fix` branch for their code changes.
+      1. The code changes should include an update to [CHANGELOG.md](/CHANGELOG.md) under the `### Security` subheader describing the issue addressed.
+   2. Once the changes are ready for review, they should be submitted as a PR to the master branch of the temporary fork.
+      1. GitHub does not currently appear to support protecting the master branch in temporary forks, so maintainers will need to enforce this process manually.
```

Comment:
Consider clarifying "protecting the master branch in temporary forks" a bit more. Does it mean any changes directly made to the master branch will be public?

Reply:
I've clarified the wording around branch protection as requiring PR reviews for merging.